### PR TITLE
bitrise 1.3.0

### PR DESCRIPTION
**Pull request by godrei**

### CODE REVIEW

File: Formula/bitrise.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
-require "language/go"
+ require "language/go"
+ require 'fileutils'
```
Comment (DomT4) [inline Formula/bitrise.rb]: This already happens automatically for formula, you don't need to include it.

File: Formula/bitrise.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
-require "language/go"
+ require "language/go"
```
Comment (DomT4) [inline Formula/bitrise.rb]: No need to indent this.

File: Formula/bitrise.rb [hunk 3]
```diff
@@ -14,36 +15,53 @@ class Bitrise < Formula
...
-    ln_s basepth, "#{basepth}/src/github.com/bitrise-io/#{pkgname}"
+  def install
+    ENV["GOPATH"] = buildpath
 
-    ENV["GOPATH"] = "#{basepth}/Godeps/_workspace:#{basepth}"
-    Language::Go.stage_deps resources, "#{basepth}/src"
-    system "go", "build", "-o", "#{bin}/#{pkgname}"
-  end
+    # Install bitrise
+    (buildpath + "src/github.com/bitrise-io/bitrise").install Dir[buildpath/"*"]
```
Comment (DomT4) [inline Formula/bitrise.rb]: `(buildpath/"src/github.com/bitrise-io/bitrise").install Dir["*"]` takes up slightly less space.

File: Formula/bitrise.rb [hunk 4]
```diff
@@ -14,36 +15,53 @@ class Bitrise < Formula
...
-  def install
+    cd "src/github.com/bitrise-io/bitrise" do
+      system "go", "build", "-o", bin/"bitrise"
+    end
+
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+
+    # Install envman
     resource("envman").stage do
-      go_install_package(Dir.pwd, "envman")
```
Comment (DomT4) [inline Formula/bitrise.rb]: I'm not sure why this couldn't be something like:
```ruby
envman_go_pth = buildpath/"src/github.com/bitrise-io/envman"
envman_go_pth.install Dir["*"]
```
Am I missing something?

**Comment by godrei** [comment]
Hi @DomT4 ,
i updated the PR based on your requests, what do you think about this version?
Thanks!

**Comment by DomT4** [comment]
LGTM.

Merged in https://github.com/Homebrew/homebrew-core/commit/a10c83bc0e39380e8b100ab80736c4fea59fe198. Thank you for your contribution to Homebrew @godrei; we appreciate it! :smiley_cat:

**Pull request closed without merging**